documentation/docs/student/quiz-training-mode.mdx

```diff
@@ -0,0 +1,204 @@
+---
+id: quiz-training
+title: Quiz Training Mode
+sidebar_label: Quiz Training
+---
+
+import trainingTab from "./assets/quiz-training-mode/quiz-training-tab.png";
+import scorecard from "./assets/quiz-training-mode/scorecard.png";
+import unratedNotification from "./assets/quiz-training-mode/unrated-notification.png";
+import description from "./assets/quiz-training-mode/description.png";
+import leaderboard from "./assets/quiz-training-mode/leaderboard.png";
+import userSettings from "./assets/quiz-training-mode/user-settings.png";
+import userLeaderboardSettings from "./assets/quiz-training-mode/user-leaderboard-settings.png";
+import startTrainingButton from "./assets/quiz-training-mode/start-training-button.png";
+import viewDescriptionButton from "./assets/quiz-training-mode/view-description-button.png";
+import Callout from "../../src/components/Callout/Callout";
+import {CalloutVariant} from "../../src/components/Callout/Callout.types";
+import Image, {ImageSize} from "../../src/components/Image/Image";
+import {Footnote, FootnoteText} from "../../src/components/Footnote/Footnote";
```

Comment:
There seems to be an issue with the imports 😅 

<img width="1374" height="370" alt="Image" src="https://github.com/user-attachments/assets/5f888b3d-40b9-4537-b2cf-6b6c002aecd4" />